### PR TITLE
XFRM: 207 added a node != null check

### PR DESCRIPTION
**Pull request by aayushshah15**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/xforms/XformBuilder.java [hunk 1]
```diff
@@ -529,6 +529,9 @@ public static void setNodeValue(Element node, String value) {
 		if (value == null)
 			value = "";
 		
+    if (node == null)
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/xforms/XformBuilder.java]: Can you improve the indention?

Comment (aayushshah15) [inline api/src/main/java/org/openmrs/module/xforms/XformBuilder.java]: Sure thing

File: api/src/main/java/org/openmrs/module/xforms/XformBuilder.java [hunk 2]
```diff
@@ -529,6 +529,9 @@ public static void setNodeValue(Element node, String value) {
 		if (value == null)
 			value = "";
 		
+		if (node == null)
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/xforms/XformBuilder.java]: Thinking about this again, i think throwing an illegal argument exception may be better than just silently ignoring it. What do you think? :)

Comment (aayushshah15) [inline api/src/main/java/org/openmrs/module/xforms/XformBuilder.java]: Alright, I'll make the change

**Pull request merged**